bloom-housing/bloom
# feat: export lottery modal

This PR addresses #4048


## Description

Adds only the export modal piece from the ticket. This PR is based off of the PR that adds the run modals.

## How Can This Be Tested/Reviewed?

With the toggle on on a closed listing with a lottery, after clicking through the run lottery button and modal, clicking on export should open the export modal.

Note that the Figma is no longer accurate. The confirmed copy and design is [here](https://docs.google.com/document/d/130OuxoQnKnukoVr_SPQq57A0JZ4vycdcS9q0EX2Gkso/edit#heading=h.btie0bci2vik) - the copy in text is the confirmed copy, not the copy in the screenshots.

This is from the Figma, but the copy should remove the preference wording if the listing has no preferences.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
